pkg/scheduler/framework/runtime/framework.go

```diff
@@ -734,6 +734,7 @@ func (f *frameworkImpl) RunFilterPlugins(
 		if state.SkipFilterPlugins.Has(pl.Name()) {
 			continue
 		}
+		metrics.PluginEvaluationTotal.WithLabelValues(pl.Name(), Filter).Inc()
```

Comment:
Let's add PreFilter too? Just in case some plugins only implement PreFilter.

Reply:
Let's start with preFilter/filter.
@kidddddddddddddddddddddd will work on the prescore/score.
https://github.com/kubernetes/kubernetes/issues/110643#issuecomment-1430051794

I'm not sure if it's worth implementing on other extension points for now because it's other extension points don't have `Skip` feature. I'd say we can support other extension points when we support `Skip` on them.